tools/interactive/interactivetool_ilastik.xml

```diff
@@ -27,16 +27,16 @@
         #end if
         export HOME=\$PWD &&
         ## Create a directory where the app user has access
-        mkdir -p ./output &&
-        chown 1000:1000 ./output/ &&
+        mkdir -p ./output ./zarr_output &&
+        chown 1000:1000 ./output/ ./zarr_output &&
         ## Make a copy of the existing project if exists
         #if str($input_type.existing) == "existing":
             cp '$input_type.project' ./output/MyProject.ilp &&
         #end if
 
         ## Link input images to current working directory
         #for input in $infiles:
-            ln -s '$input' ./'$input.element_identifier'.tif &&
+            ln -s '$input' ./'$input.element_identifier' &&
```

Comment:
the exention is not needed?

Mybe add .tif or .h5? Or is this not useful for the user anymore?